src/main/java/core/basesyntax/ReadWriteList.java

```diff
@@ -10,16 +11,32 @@ public class ReadWriteList<E> {
     private ReadWriteLock lock = new ReentrantReadWriteLock();
 
     public void add(E element) {
-        // write your code here
+        Lock writeLock = lock.writeLock();
```

Comment:
Creating a variable for lock is not necessary actually, as there is only one instance of read and write locks per ReentrantReadWriteLock instance (or make them instance variables)